ERROR - 2023-07-16 09:40:42 --> Error connecting to the database: mysqli_sql_exception: Access denied for user ''@'localhost' (using password: NO) in C:\xampp\htdocs\prakwebprog\system\Database\MySQLi\Connection.php:174
Stack trace:
#0 C:\xampp\htdocs\prakwebprog\system\Database\MySQLi\Connection.php(174): mysqli->real_connect('localhost', '', '', '', 3306, '', 0)
#1 C:\xampp\htdocs\prakwebprog\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\prakwebprog\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\prakwebprog\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp\htdocs\prakwebprog\system\Model.php(241): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\prakwebprog\system\BaseModel.php(618): CodeIgniter\Model->doFindAll(0, 0)
#6 C:\xampp\htdocs\prakwebprog\app\Controllers\Books.php(13): CodeIgniter\BaseModel->findAll()
#7 C:\xampp\htdocs\prakwebprog\system\CodeIgniter.php(934): App\Controllers\Books->index()
#8 C:\xampp\htdocs\prakwebprog\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Books))
#9 C:\xampp\htdocs\prakwebprog\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\prakwebprog\public\index.php(67): CodeIgniter\CodeIgniter->run()
#11 C:\xampp\htdocs\prakwebprog\system\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs...')
#12 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Access denied for user ''@'localhost' (using password: NO) in C:\xampp\htdocs\prakwebprog\system\Database\MySQLi\Connection.php:218
Stack trace:
#0 C:\xampp\htdocs\prakwebprog\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\prakwebprog\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\prakwebprog\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 C:\xampp\htdocs\prakwebprog\system\Model.php(241): CodeIgniter\Database\BaseBuilder->get()
#4 C:\xampp\htdocs\prakwebprog\system\BaseModel.php(618): CodeIgniter\Model->doFindAll(0, 0)
#5 C:\xampp\htdocs\prakwebprog\app\Controllers\Books.php(13): CodeIgniter\BaseModel->findAll()
#6 C:\xampp\htdocs\prakwebprog\system\CodeIgniter.php(934): App\Controllers\Books->index()
#7 C:\xampp\htdocs\prakwebprog\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Books))
#8 C:\xampp\htdocs\prakwebprog\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp\htdocs\prakwebprog\public\index.php(67): CodeIgniter\CodeIgniter->run()
#10 C:\xampp\htdocs\prakwebprog\system\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs...')
#11 {main}
CRITICAL - 2023-07-16 09:40:42 --> Unable to connect to the database.
Main connection [MySQLi]: Access denied for user ''@'localhost' (using password: NO)
in SYSTEMPATH\Database\BaseConnection.php on line 427.
 1 SYSTEMPATH\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `books`', [], false)
 3 SYSTEMPATH\Model.php(241): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH\BaseModel.php(618): CodeIgniter\Model->doFindAll(0, 0)
 5 APPPATH\Controllers\Books.php(13): CodeIgniter\BaseModel->findAll()
 6 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Books->index()
 7 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Books))
 8 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs\\prakwebprog\\public\\index.php')
ERROR - 2023-07-16 09:40:42 --> Error connecting to the database: mysqli_sql_exception: Access denied for user ''@'localhost' (using password: NO) in C:\xampp\htdocs\prakwebprog\system\Database\MySQLi\Connection.php:174
Stack trace:
#0 C:\xampp\htdocs\prakwebprog\system\Database\MySQLi\Connection.php(174): mysqli->real_connect('localhost', '', '', '', 3306, '', 0)
#1 C:\xampp\htdocs\prakwebprog\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\prakwebprog\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\prakwebprog\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp\htdocs\prakwebprog\system\Model.php(241): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\prakwebprog\system\BaseModel.php(618): CodeIgniter\Model->doFindAll(0, 0)
#6 C:\xampp\htdocs\prakwebprog\app\Controllers\Books.php(13): CodeIgniter\BaseModel->findAll()
#7 C:\xampp\htdocs\prakwebprog\system\CodeIgniter.php(934): App\Controllers\Books->index()
#8 C:\xampp\htdocs\prakwebprog\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Books))
#9 C:\xampp\htdocs\prakwebprog\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\prakwebprog\public\index.php(67): CodeIgniter\CodeIgniter->run()
#11 C:\xampp\htdocs\prakwebprog\system\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs...')
#12 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Access denied for user ''@'localhost' (using password: NO) in C:\xampp\htdocs\prakwebprog\system\Database\MySQLi\Connection.php:218
Stack trace:
#0 C:\xampp\htdocs\prakwebprog\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\prakwebprog\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\prakwebprog\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 C:\xampp\htdocs\prakwebprog\system\Model.php(241): CodeIgniter\Database\BaseBuilder->get()
#4 C:\xampp\htdocs\prakwebprog\system\BaseModel.php(618): CodeIgniter\Model->doFindAll(0, 0)
#5 C:\xampp\htdocs\prakwebprog\app\Controllers\Books.php(13): CodeIgniter\BaseModel->findAll()
#6 C:\xampp\htdocs\prakwebprog\system\CodeIgniter.php(934): App\Controllers\Books->index()
#7 C:\xampp\htdocs\prakwebprog\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Books))
#8 C:\xampp\htdocs\prakwebprog\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp\htdocs\prakwebprog\public\index.php(67): CodeIgniter\CodeIgniter->run()
#10 C:\xampp\htdocs\prakwebprog\system\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs...')
#11 {main}
CRITICAL - 2023-07-16 09:40:42 --> Unable to connect to the database.
Main connection [MySQLi]: Access denied for user ''@'localhost' (using password: NO)
in SYSTEMPATH\Database\BaseConnection.php on line 427.
 1 SYSTEMPATH\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `books`', [], false)
 3 SYSTEMPATH\Model.php(241): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH\BaseModel.php(618): CodeIgniter\Model->doFindAll(0, 0)
 5 APPPATH\Controllers\Books.php(13): CodeIgniter\BaseModel->findAll()
 6 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Books->index()
 7 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Books))
 8 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs\\prakwebprog\\public\\index.php')
ERROR - 2023-07-16 09:46:51 --> Error connecting to the database: mysqli_sql_exception: Access denied for user ''@'localhost' (using password: NO) in C:\xampp\htdocs\prakwebprog\system\Database\MySQLi\Connection.php:174
Stack trace:
#0 C:\xampp\htdocs\prakwebprog\system\Database\MySQLi\Connection.php(174): mysqli->real_connect('localhost', '', '', '', 3306, '', 0)
#1 C:\xampp\htdocs\prakwebprog\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\prakwebprog\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\prakwebprog\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp\htdocs\prakwebprog\system\Model.php(241): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\prakwebprog\system\BaseModel.php(618): CodeIgniter\Model->doFindAll(0, 0)
#6 C:\xampp\htdocs\prakwebprog\app\Controllers\Books.php(13): CodeIgniter\BaseModel->findAll()
#7 C:\xampp\htdocs\prakwebprog\system\CodeIgniter.php(934): App\Controllers\Books->index()
#8 C:\xampp\htdocs\prakwebprog\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Books))
#9 C:\xampp\htdocs\prakwebprog\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\prakwebprog\public\index.php(67): CodeIgniter\CodeIgniter->run()
#11 C:\xampp\htdocs\prakwebprog\system\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs...')
#12 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Access denied for user ''@'localhost' (using password: NO) in C:\xampp\htdocs\prakwebprog\system\Database\MySQLi\Connection.php:218
Stack trace:
#0 C:\xampp\htdocs\prakwebprog\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\prakwebprog\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\prakwebprog\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 C:\xampp\htdocs\prakwebprog\system\Model.php(241): CodeIgniter\Database\BaseBuilder->get()
#4 C:\xampp\htdocs\prakwebprog\system\BaseModel.php(618): CodeIgniter\Model->doFindAll(0, 0)
#5 C:\xampp\htdocs\prakwebprog\app\Controllers\Books.php(13): CodeIgniter\BaseModel->findAll()
#6 C:\xampp\htdocs\prakwebprog\system\CodeIgniter.php(934): App\Controllers\Books->index()
#7 C:\xampp\htdocs\prakwebprog\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Books))
#8 C:\xampp\htdocs\prakwebprog\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp\htdocs\prakwebprog\public\index.php(67): CodeIgniter\CodeIgniter->run()
#10 C:\xampp\htdocs\prakwebprog\system\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs...')
#11 {main}
CRITICAL - 2023-07-16 09:46:51 --> Unable to connect to the database.
Main connection [MySQLi]: Access denied for user ''@'localhost' (using password: NO)
in SYSTEMPATH\Database\BaseConnection.php on line 427.
 1 SYSTEMPATH\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `books`', [], false)
 3 SYSTEMPATH\Model.php(241): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH\BaseModel.php(618): CodeIgniter\Model->doFindAll(0, 0)
 5 APPPATH\Controllers\Books.php(13): CodeIgniter\BaseModel->findAll()
 6 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Books->index()
 7 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Books))
 8 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs\\prakwebprog\\public\\index.php')
ERROR - 2023-07-16 09:50:33 --> Error connecting to the database: mysqli_sql_exception: Access denied for user ''@'localhost' (using password: NO) in C:\xampp\htdocs\prakwebprog\system\Database\MySQLi\Connection.php:174
Stack trace:
#0 C:\xampp\htdocs\prakwebprog\system\Database\MySQLi\Connection.php(174): mysqli->real_connect('localhost', '', '', '', 3306, '', 0)
#1 C:\xampp\htdocs\prakwebprog\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\prakwebprog\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\prakwebprog\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp\htdocs\prakwebprog\system\Model.php(241): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\prakwebprog\system\BaseModel.php(618): CodeIgniter\Model->doFindAll(0, 0)
#6 C:\xampp\htdocs\prakwebprog\app\Controllers\Books.php(13): CodeIgniter\BaseModel->findAll()
#7 C:\xampp\htdocs\prakwebprog\system\CodeIgniter.php(934): App\Controllers\Books->index()
#8 C:\xampp\htdocs\prakwebprog\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Books))
#9 C:\xampp\htdocs\prakwebprog\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\prakwebprog\public\index.php(67): CodeIgniter\CodeIgniter->run()
#11 C:\xampp\htdocs\prakwebprog\system\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs...')
#12 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Access denied for user ''@'localhost' (using password: NO) in C:\xampp\htdocs\prakwebprog\system\Database\MySQLi\Connection.php:218
Stack trace:
#0 C:\xampp\htdocs\prakwebprog\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\prakwebprog\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\prakwebprog\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 C:\xampp\htdocs\prakwebprog\system\Model.php(241): CodeIgniter\Database\BaseBuilder->get()
#4 C:\xampp\htdocs\prakwebprog\system\BaseModel.php(618): CodeIgniter\Model->doFindAll(0, 0)
#5 C:\xampp\htdocs\prakwebprog\app\Controllers\Books.php(13): CodeIgniter\BaseModel->findAll()
#6 C:\xampp\htdocs\prakwebprog\system\CodeIgniter.php(934): App\Controllers\Books->index()
#7 C:\xampp\htdocs\prakwebprog\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Books))
#8 C:\xampp\htdocs\prakwebprog\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp\htdocs\prakwebprog\public\index.php(67): CodeIgniter\CodeIgniter->run()
#10 C:\xampp\htdocs\prakwebprog\system\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs...')
#11 {main}
CRITICAL - 2023-07-16 09:50:33 --> Unable to connect to the database.
Main connection [MySQLi]: Access denied for user ''@'localhost' (using password: NO)
in SYSTEMPATH\Database\BaseConnection.php on line 427.
 1 SYSTEMPATH\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `books`', [], false)
 3 SYSTEMPATH\Model.php(241): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH\BaseModel.php(618): CodeIgniter\Model->doFindAll(0, 0)
 5 APPPATH\Controllers\Books.php(13): CodeIgniter\BaseModel->findAll()
 6 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Books->index()
 7 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Books))
 8 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs\\prakwebprog\\public\\index.php')
ERROR - 2023-07-16 09:51:07 --> Error connecting to the database: mysqli_sql_exception: Access denied for user ''@'localhost' (using password: NO) in C:\xampp\htdocs\prakwebprog\system\Database\MySQLi\Connection.php:174
Stack trace:
#0 C:\xampp\htdocs\prakwebprog\system\Database\MySQLi\Connection.php(174): mysqli->real_connect('localhost', '', '', '', 3306, '', 0)
#1 C:\xampp\htdocs\prakwebprog\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\prakwebprog\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\prakwebprog\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp\htdocs\prakwebprog\system\Model.php(241): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\prakwebprog\system\BaseModel.php(618): CodeIgniter\Model->doFindAll(0, 0)
#6 C:\xampp\htdocs\prakwebprog\app\Controllers\Books.php(13): CodeIgniter\BaseModel->findAll()
#7 C:\xampp\htdocs\prakwebprog\system\CodeIgniter.php(934): App\Controllers\Books->index()
#8 C:\xampp\htdocs\prakwebprog\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Books))
#9 C:\xampp\htdocs\prakwebprog\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\prakwebprog\public\index.php(67): CodeIgniter\CodeIgniter->run()
#11 C:\xampp\htdocs\prakwebprog\system\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs...')
#12 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Access denied for user ''@'localhost' (using password: NO) in C:\xampp\htdocs\prakwebprog\system\Database\MySQLi\Connection.php:218
Stack trace:
#0 C:\xampp\htdocs\prakwebprog\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\prakwebprog\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\prakwebprog\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 C:\xampp\htdocs\prakwebprog\system\Model.php(241): CodeIgniter\Database\BaseBuilder->get()
#4 C:\xampp\htdocs\prakwebprog\system\BaseModel.php(618): CodeIgniter\Model->doFindAll(0, 0)
#5 C:\xampp\htdocs\prakwebprog\app\Controllers\Books.php(13): CodeIgniter\BaseModel->findAll()
#6 C:\xampp\htdocs\prakwebprog\system\CodeIgniter.php(934): App\Controllers\Books->index()
#7 C:\xampp\htdocs\prakwebprog\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Books))
#8 C:\xampp\htdocs\prakwebprog\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp\htdocs\prakwebprog\public\index.php(67): CodeIgniter\CodeIgniter->run()
#10 C:\xampp\htdocs\prakwebprog\system\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs...')
#11 {main}
CRITICAL - 2023-07-16 09:51:07 --> Unable to connect to the database.
Main connection [MySQLi]: Access denied for user ''@'localhost' (using password: NO)
in SYSTEMPATH\Database\BaseConnection.php on line 427.
 1 SYSTEMPATH\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `books`', [], false)
 3 SYSTEMPATH\Model.php(241): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH\BaseModel.php(618): CodeIgniter\Model->doFindAll(0, 0)
 5 APPPATH\Controllers\Books.php(13): CodeIgniter\BaseModel->findAll()
 6 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Books->index()
 7 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Books))
 8 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs\\prakwebprog\\public\\index.php')
ERROR - 2023-07-16 09:51:52 --> Error connecting to the database: mysqli_sql_exception: Access denied for user ''@'localhost' (using password: NO) in C:\xampp\htdocs\prakwebprog\system\Database\MySQLi\Connection.php:174
Stack trace:
#0 C:\xampp\htdocs\prakwebprog\system\Database\MySQLi\Connection.php(174): mysqli->real_connect('localhost', '', '', '', 3306, '', 0)
#1 C:\xampp\htdocs\prakwebprog\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\prakwebprog\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\prakwebprog\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp\htdocs\prakwebprog\system\Model.php(241): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\prakwebprog\system\BaseModel.php(618): CodeIgniter\Model->doFindAll(0, 0)
#6 C:\xampp\htdocs\prakwebprog\app\Controllers\Books.php(13): CodeIgniter\BaseModel->findAll()
#7 C:\xampp\htdocs\prakwebprog\system\CodeIgniter.php(934): App\Controllers\Books->index()
#8 C:\xampp\htdocs\prakwebprog\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Books))
#9 C:\xampp\htdocs\prakwebprog\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\prakwebprog\public\index.php(67): CodeIgniter\CodeIgniter->run()
#11 C:\xampp\htdocs\prakwebprog\system\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs...')
#12 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Access denied for user ''@'localhost' (using password: NO) in C:\xampp\htdocs\prakwebprog\system\Database\MySQLi\Connection.php:218
Stack trace:
#0 C:\xampp\htdocs\prakwebprog\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\prakwebprog\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\prakwebprog\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 C:\xampp\htdocs\prakwebprog\system\Model.php(241): CodeIgniter\Database\BaseBuilder->get()
#4 C:\xampp\htdocs\prakwebprog\system\BaseModel.php(618): CodeIgniter\Model->doFindAll(0, 0)
#5 C:\xampp\htdocs\prakwebprog\app\Controllers\Books.php(13): CodeIgniter\BaseModel->findAll()
#6 C:\xampp\htdocs\prakwebprog\system\CodeIgniter.php(934): App\Controllers\Books->index()
#7 C:\xampp\htdocs\prakwebprog\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Books))
#8 C:\xampp\htdocs\prakwebprog\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp\htdocs\prakwebprog\public\index.php(67): CodeIgniter\CodeIgniter->run()
#10 C:\xampp\htdocs\prakwebprog\system\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs...')
#11 {main}
CRITICAL - 2023-07-16 09:51:52 --> Unable to connect to the database.
Main connection [MySQLi]: Access denied for user ''@'localhost' (using password: NO)
in SYSTEMPATH\Database\BaseConnection.php on line 427.
 1 SYSTEMPATH\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `books`', [], false)
 3 SYSTEMPATH\Model.php(241): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH\BaseModel.php(618): CodeIgniter\Model->doFindAll(0, 0)
 5 APPPATH\Controllers\Books.php(13): CodeIgniter\BaseModel->findAll()
 6 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Books->index()
 7 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Books))
 8 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs\\prakwebprog\\public\\index.php')
ERROR - 2023-07-16 09:51:53 --> Error connecting to the database: mysqli_sql_exception: Access denied for user ''@'localhost' (using password: NO) in C:\xampp\htdocs\prakwebprog\system\Database\MySQLi\Connection.php:174
Stack trace:
#0 C:\xampp\htdocs\prakwebprog\system\Database\MySQLi\Connection.php(174): mysqli->real_connect('localhost', '', '', '', 3306, '', 0)
#1 C:\xampp\htdocs\prakwebprog\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\prakwebprog\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\prakwebprog\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp\htdocs\prakwebprog\system\Model.php(241): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\prakwebprog\system\BaseModel.php(618): CodeIgniter\Model->doFindAll(0, 0)
#6 C:\xampp\htdocs\prakwebprog\app\Controllers\Books.php(13): CodeIgniter\BaseModel->findAll()
#7 C:\xampp\htdocs\prakwebprog\system\CodeIgniter.php(934): App\Controllers\Books->index()
#8 C:\xampp\htdocs\prakwebprog\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Books))
#9 C:\xampp\htdocs\prakwebprog\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\prakwebprog\public\index.php(67): CodeIgniter\CodeIgniter->run()
#11 C:\xampp\htdocs\prakwebprog\system\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs...')
#12 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Access denied for user ''@'localhost' (using password: NO) in C:\xampp\htdocs\prakwebprog\system\Database\MySQLi\Connection.php:218
Stack trace:
#0 C:\xampp\htdocs\prakwebprog\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\prakwebprog\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\prakwebprog\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 C:\xampp\htdocs\prakwebprog\system\Model.php(241): CodeIgniter\Database\BaseBuilder->get()
#4 C:\xampp\htdocs\prakwebprog\system\BaseModel.php(618): CodeIgniter\Model->doFindAll(0, 0)
#5 C:\xampp\htdocs\prakwebprog\app\Controllers\Books.php(13): CodeIgniter\BaseModel->findAll()
#6 C:\xampp\htdocs\prakwebprog\system\CodeIgniter.php(934): App\Controllers\Books->index()
#7 C:\xampp\htdocs\prakwebprog\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Books))
#8 C:\xampp\htdocs\prakwebprog\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp\htdocs\prakwebprog\public\index.php(67): CodeIgniter\CodeIgniter->run()
#10 C:\xampp\htdocs\prakwebprog\system\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs...')
#11 {main}
CRITICAL - 2023-07-16 09:51:53 --> Unable to connect to the database.
Main connection [MySQLi]: Access denied for user ''@'localhost' (using password: NO)
in SYSTEMPATH\Database\BaseConnection.php on line 427.
 1 SYSTEMPATH\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `books`', [], false)
 3 SYSTEMPATH\Model.php(241): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH\BaseModel.php(618): CodeIgniter\Model->doFindAll(0, 0)
 5 APPPATH\Controllers\Books.php(13): CodeIgniter\BaseModel->findAll()
 6 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Books->index()
 7 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Books))
 8 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Commands\Server\rewrite.php(47): require_once('C:\\xampp\\htdocs\\prakwebprog\\public\\index.php')
